CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
   * `cortex_ingester_received_files`
   * `cortex_ingester_received_bytes_total`
   * `cortex_ingester_sent_bytes_total`
+* [CHANGE] Use custom histogram Bucket for variable in series store`indexLookupsPerQuery`#3021
```

Comment:
When you write a CHANGELOG entry try to always express the change as understandeable from a user who has never seen Cortex internals. In this case it's better to mention the metric name.

```suggestion
* [CHANGE] The buckets for the `cortex_chunk_store_index_lookups_per_query` have been changed to 1, 2, 4, 8, 16. #3021
```

Reply:
Thanks @pracucci . I will update this. :)